onyo/cli/get.py

```diff
@@ -115,6 +115,17 @@
             One can sort by keys that are not in the output.
         """
```

Comment:
GitHub doesn't let me comment on the lines I want to.

Above, please:
- in `"match"`, please remove the word "assets" from" Criteria to match assets in the form ``KEY=VALUE``, ..."
- in `"include"` please replace "Assets or directories" with "Items".
- in `"exclude"` please replace "Assets or directories" with "Items".

Reply:
Re include/exclude: I'll make it `Paths` instead, since this is what the user needs to specify there. Not some "item identifier" but paths.

Reply:
Yes. That's even better. :-)